router/internal/graphiql/playground.go

```diff
@@ -7,6 +7,6 @@ import (
 //go:embed graphiql.html
 var s string
 
-func GetGraphiqlPlaygroundHTML() string {
+func PlaygroundHTML() string {
```

Comment:
Subscriptions in the Studio don't work

![image](https://github.com/wundergraph/cosmo/assets/1764424/7ca87957-f5be-486a-a2f8-eeaa3da2e5fd)

Reply:
Does your router already support them?

Reply:
Yes, @fiam just forgot to update the playground to support subscriptions.